e2e/README.md

```diff
@@ -40,6 +40,12 @@ Both command do the same, use the saucelabs as a driver, but you get the results
 
 ### pictures
 
+To see more information about the failing tests you can use the 'DEBUG' environment variable.
+
+- `DEBUG=test CI=true npm run test:e2e`
```

Comment:
why is the `CI` needed?

Reply:
i would remove completely the `CI` variable from the README

Reply:
Ok, i will do it in a separated PR.